.github/scripts/end2end/deploy-zenko.sh

```diff
@@ -17,6 +17,7 @@ export ZENKO_STS_INGRESS=${ZENKO_STS_INGRESS:-'sts.zenko.local'}
 export ZENKO_MANAGEMENT_INGRESS=${ZENKO_MANAGEMENT_INGRESS:-'management.zenko.local'}
 export ZENKO_S3_INGRESS=${ZENKO_S3_INGRESS:-'s3.zenko.local'}
 export ZENKO_UI_INGRESS=${ZENKO_UI_INGRESS:-'ui.zenko.local'}
+export ZENKO_SUR_API_INGRESS=${ZENKO_SUR_API_INGRESS:-'utilization.zenko.local'}
```

Comment:
nit: just `ZENKO_SUR_INGRESS`

```suggestion
export ZENKO_SUR_INGRESS=${ZENKO_SUR_API_INGRESS:-'utilization.zenko.local'}
```

(don't forget to update references below...)